src/dimension.c

```diff
@@ -308,7 +308,11 @@ hyperspace_calculate_point(Hyperspace *hs, HeapTuple tuple, TupleDesc tupdesc)
 			datum = heap_getattr(tuple, d->column_attno, tupdesc, &isnull);
 
 			if (isnull)
-				elog(ERROR, "Time attribute not found in tuple");
+				ereport(ERROR,
+						(errcode(ERRCODE_NOT_NULL_VIOLATION),
+						 errmsg("null value in column \"%s\" violates NON NULL constraint",
```

Comment:
Shouldn't it be "NOT NULL", rather than "NON NULL"?

Reply:
yes